spec/spec_helper.rb

```diff
@@ -26,7 +26,14 @@
 require 'capybara/rspec'
 require 'capybara/rails'
 
+# Devise helpers
+require 'devise'
+
+Dir[Rails.root.join("spec/support/**/*.rb")].each { |f| require f }
```

Comment:
Prefer single-quoted strings when you don't need string interpolation or special symbols.